app/src/main/java/com/space/quizapp/presentation/ui/points/QuizPointsViewModel.kt

```diff
@@ -1,7 +1,20 @@
 package com.space.quizapp.presentation.ui.points
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import com.space.quizapp.domain.usecase.current_user.clear.ClearUserSessionUseCase
+import com.space.quizapp.domain.usecase.current_user.get.GetUserSessionUseCase
+import com.space.quizapp.utils.extensions.viewModelScope
 
-class QuizPointsViewModel : ViewModel() {
-    // TODO: Implement the ViewModel
+class QuizPointsViewModel(
+    private val clearUserSessionUseCase: ClearUserSessionUseCase
+) : ViewModel() {
+
+    fun clearUserSession(completion: () -> Unit){
+        viewModelScope {
+            clearUserSessionUseCase.invoke()
+            completion.invoke()
+        }
```

Comment:
same here you dont need lambda

Reply:
აქ გასწორებულია https://github.com/iTarley/QuizApp-SpaceIntern/pull/5